### PR TITLE
WIP to get simulator to work with an actual application that uses paygate

### PR DESCRIPTION
**Pull request by trojanc**
This PR is still a work in progress.
Updates thus far

- Update clarity icons and fix build issues
- Angular proxy config to test UI with spring backend
- Package UI with services to allow single jar executable with all packaged (could be a 3rd project later)
- Restructure UI slightly to keep track of paygateId
- Additional services to get transaction that is busy from the UI

### CODE REVIEW

File: ui/angular.json [hunk 1]
```diff
@@ -24,13 +24,9 @@
             ],
             "styles": [
               "src/styles.css",
-              "node_modules/bootstrap/dist/css/bootstrap.min.css",
```
Comment (dirkjlouw) [inline ui/angular.json]: Don't we have to use clarity styles? Just use the module as you import it into the app.module.ts?

**Comment by dirkjlouw** [comment]
Work looks good Charl

**Comment by trojanc** [comment]
This PR is now complete. I've tested it with VSSLite. A PR for VSSlite is also on its way to allow more configuration options for paygate integration and to allow it to point to another service

**Comment by dirkjlouw** [comment]
Jis, can we have a quick demo of the code maybe. would like to see.

File: ui/angular.json (hunk 1 above)
Comment (trojanc) [reply ui/angular.json]: Yes i found that this isn't necessary when using clarity-angular

**Pull request merged**